manifests/4.1/elasticsearch-operator.v4.1.0.clusterserviceversion.yaml

```diff
@@ -68,6 +69,10 @@ spec:
 
   keywords: ['elasticsearch', 'jaeger']
 
+  replaces: elasticsearch-operator.v4.1.0
```

Comment:
I don't believe this is necessary for z streams, only for something like 4.1 -> 4.2

Reply:
@ecordell @jwforres can you confirm?